### ISSUE TITLE
Dead project ?

### ISSUE THREAD

**Issue by AkechiShiro** [leetronics/infnoise#33]
Hi, is this project still maintained, last commit was 6 months ago

**Comment by felixd** [comment]
Looks like. 
Are you looking for something? ;)

**Comment by AkechiShiro** [comment]
Is there any @felixd other project in the same subject but actively maintained ? for a hardware TRNG ?

**Comment by felixd** [comment]
@AkechiShiro I haven't found one + this project seems to be best option to start with even though it's not "actively" maintained. 

Everything is well described and documented. 

What I am personally doing as well is using my CCTV cameras as another entropy source.
If You want to test it I can upload You some random data from it for evaluation.

**Comment by AkechiShiro** [comment]
Okay so this is what's best today as far as you know, it's okay, I was just wondering if you knew something else. Thanks for proposing but I was just curious

**Comment by Strykar** [comment]
@AkechiShiro - https://github.com/atoponce/awesome-hwrng

**Comment by Strykar** [comment]
@wil-langford Yes, I received mine a few days later from Germany than expected, but arrived within two weeks to Asia. You should get yours, might take longer is all.

**Comment by TrustExecutor** [comment]
> @Strykar Did you get the hardware you ordered from Leetronics? I ask because I'm not getting any sort of status updates on my own order and it directly relates to whether or not the project is dead.
> 
> In any case, thanks for the link to `awesome-hwrng`.

Did you get yours eventually? I ordered over a month ago (within EU) and haven't heard anything, and nobody reply to status update enqueries.

**Comment by wil-langford** [comment]
I ordered it on March 27 and an envelope was delivered to Oregon, USA on April 25.  I haven't bothered to open it yet as I no longer trust whatever is in there.  If I need randomness I'll likely just make do with the built-in hardware options on some of my machines.

**Comment by AkechiShiro** [comment]
> I ordered it on March 27 and an envelope was delivered to Oregon, USA on April 25.  I haven't bothered to open it yet as I no longer trust whatever is in there.  If I need randomness I'll likely just make do with the built-in hardware options on some of my machines.

I don't understand why you're not trusting some thing you bought, you should've made one yourself I guess.

**Comment by wil-langford** [comment]
We're well outside the scope of this issue, but essentially you're correct about the delay suggesting tampering being the source of my (not entirely rational) hesitancy.

**Comment by manuel-domke** [comment]
@AkechiShiro @wil-langford 

I'd like to point out that the infinite noise was specifically designed in a manner that should make any kind of tampering pretty much impossible. That's mostly because of the use of an off-the-shelf chip (the device has *no firmware*) and the health monitoring built into the open-source software.

**Comment by Strykar** [comment]
Project is so dead, that it recently acquired:
A kernel module - https://github.com/waywardgeek/infnoise/pull/111
An OpenSSL 3.x provider - https://github.com/Strykar/infnoise-provider
